redash/permissions.py

```diff
@@ -13,6 +13,24 @@
 
 ACCESS_TYPES = (ACCESS_TYPE_VIEW, ACCESS_TYPE_MODIFY, ACCESS_TYPE_DELETE)
 
+def has_access_torun(object_groups, user, need_view_only):
+    if 'admin' in user.permissions:
+        return True
+
+    matching_groups = set(object_groups.keys()).intersection(user.group_ids)
+
+    if not matching_groups:
+        return False
+
+    # ADD control if only default group overlapping
+    if len(matching_groups) == 1 and str(list(matching_groups)[0]) == '2':
```

Comment:
Can we be sure that at all redash installations `default` group has the id 2? We should check